### ISSUE TITLE
Simpliciter crash when changing Grid number

### ISSUE THREAD

**Issue by cschol** [nysthi/nysthi#259]
NYSTHI version: 1.0.8
Rack version tested: 1.1.6 (and latest dev version)

Steps to reproduce:
- Add Simpliciter
- Load WAV file
- Increase the Grid number with the mouse
- Rack will crash with Segmentation Fault

![simpliciter](https://user-images.githubusercontent.com/624427/69765879-e4082f00-1132-11ea-827f-9c5644d31800.png)

Backtrace:
```
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
bt
Core was generated by `./Rack -d'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007f924f15612d in Simpliciter::process(rack::engine::Module::ProcessArgs const&) () from ./plugins/NYSTHI/plugin.so
[Current thread is 1 (Thread 0x7f9240d96700 (LWP 6820))]
(gdb) bt
#0  0x00007f924f15612d in Simpliciter::process(rack::engine::Module::ProcessArgs const&) () at ./plugins/NYSTHI/plugin.so
#1  0x00005632e1fefd35 in rack::engine::Engine_stepModules(rack::engine::Engine*, int) (that=that@entry=0x5632e4593bf0, threadId=0) at src/engine/Engine.cpp:272
#2  0x00005632e1ff1c69 in rack::engine::Engine_step (that=0x5632e4593bf0) at src/engine/Engine.cpp:346
#3  0x00005632e1ff1c69 in rack::engine::Engine_run (that=0x5632e4593bf0) at src/engine/Engine.cpp:448
#4  0x00005632e1ff1c69 in rack::engine::Engine::<lambda()>::operator() (__closure=<optimized out>) at src/engine/Engine.cpp:482
#5  0x00005632e1ff1c69 in std::__invoke_impl<void, rack::engine::Engine::start()::<lambda()> > (__f=...) at /usr/include/c++/7/bits/invoke.h:60
#6  0x00005632e1ff1c69 in std::__invoke<rack::engine::Engine::start()::<lambda()> > (__fn=...) at /usr/include/c++/7/bits/invoke.h:95
#7  0x00005632e1ff1c69 in std::thread::_Invoker<std::tuple<rack::engine::Engine::start()::<lambda()> > >::_M_invoke<0> (this=<optimized out>) at /usr/include/c++/7/thread:234
#8  0x00005632e1ff1c69 in std::thread::_Invoker<std::tuple<rack::engine::Engine::start()::<lambda()> > >::operator() (this=<optimized out>) at /usr/include/c++/7/thread:243
#9  0x00005632e1ff1c69 in std::thread::_State_impl<std::thread::_Invoker<std::tuple<rack::engine::Engine::start()::<lambda()> > > >::_M_run(void) (this=<optimized out>) at /usr/include/c++/7/thread:186
#10 0x00007f925c58366f in  () at /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#11 0x00007f925ded26db in start_thread (arg=0x7f9240d96700) at pthread_create.c:463
#12 0x00007f925bc4088f in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
```

**Comment by nysthi** [comment]
probably I solved this issue in 1.0.9
still to come out...

**Comment by nysthi** [comment]
please recheck with 1.0.9, thank you !

**Comment by cschol** [comment]
Confirmed to be fixed in 1.0.9. Thank you!
